.travis.yml

```diff
@@ -14,8 +14,6 @@ sudo: false
 
```

Comment:
I'm a bit confused, which PR is meant to handle this? This same issue is being handled by #3. Could you please clarify on this?

Reply:
Travis seems to behave differently on all builds
on my machine i have use 'create-settings' , someone else has to use 'create_settings' with an underscore and sometimes i have to use 'create_settings' it really confusing actually

Reply:
@sir3n-sn `create-settings` is also what works on mine. I think we should stick with that unless this is also causing other issues.

Reply:
BTW this PR should only fix linting errors, setting up `travis.yml` is being handled in #3

Reply:
@collin5 Trying to configure Travis was done because we needed to collaborate on setting it up, and to make sure that our linted code was also passing on Travis

Reply:
@philophilo Ok, so I will be merging #3 in a few, so all you'll have to do is rebase this with `develop` and we won't need to setup travis in this PR.